components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -36,25 +36,25 @@ namespace scream {
         FieldLayout scalar2d_swband_layout { {COL,SWBND}, {m_ncol,m_nswbands} };
 
         // Set required (input) fields here
-        m_required_fields.emplace("pmid" , scalar3d_layout_mid, Pa, grid->name());
+        m_required_fields.emplace("p_mid" , scalar3d_layout_mid, Pa, grid->name());
         m_required_fields.emplace("pint", scalar3d_layout_int, Pa, grid->name());
```

Comment:
Is it too much to ask to change `pint` and `tint` to `p_int` and `T_int` for consistency with the change in `pmid` and `tmid`?

Reply:
Great suggestion.  I can put that into this PR

Reply:
There is a `pint` in shoc as well.

Reply:
Being Friday, "pint"s don't sound that bad after all.. ;)

Reply:
p_int is supposed to be a calculated rather than a FM variable so why is it showing up in the FM at all?

Reply:
I think the idea is to do thinks in pieces.  We don't yet have the infrastructure to calculate pint.  Once that is in we will remove pint from the FM.  So from one perspective, maybe we don't change these now because they may be replaced/removed in the future (?).  But we should keep them in the FM at least, until we are ready to calculate them.

Reply:
@PeterCaldwell I think this PR is only changing var names, to make sure all atm procs agree on names. The work to ensure atm procs only pass aroudn vars that are listed on the confluence FM page will come after this (in several separate PRs).

Reply:
So, if the plan is too eventually replace these variables with local variables that are derived, would it make sense to go ahead and merge this PR now and take care of the variable names when we get to that point?

Reply:
Yeah, the explanation of creating the function for p_int in a future PR makes sense. I just don't think it's worth worrying about p_int vs pint if we're going to delete it eventually anyways. Merge away.